sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/AbstractStorageTest.java

```diff
@@ -91,6 +92,12 @@
 
     static final String ETAG = "06000da6-0000-0000-0000-5c7093c30000";
 
+    static final long TIMESTAMP_TODAY = System.currentTimeMillis();
+
+    static final long TIMESTAMP_TOMORROW = System.currentTimeMillis() + BaseOptions.DEFAULT_EXPIRATION_IN_SECONDS;
```

Comment:
If it was SDK I would tell to use Calendar for adding or removing days, but given it's a unit test it's ok to be inaccurate for that purpose. Just afraid it will encourage that in other places. Maybe rename to past and future?

Reply:
Good point, fixed.